lib/plugin/instance.rb

```diff
@@ -130,6 +135,14 @@ def full_admin_route
       end
   end
 
+  def settings?
```

Comment:
```suggestion
  def any_settings?
```

I think this is slightly clearer, but feel free to disagree :)

Reply:
I don't hate it. 🙂